var/spack/repos/builtin/packages/py-dask/package.py

```diff
@@ -120,6 +124,7 @@ class PyDask(PythonPackage):
     # Requirements for dask.diagnostics
     depends_on('py-bokeh',              type=('build', 'run'), when='+diagnostics')
     depends_on('py-bokeh@1.0.0:',       type=('build', 'run'), when='@2.0.0: +diagnostics')
+    depends_on('py-bokeh@1.0.0:1.999,2.0.1:', type=('build', 'run'), when='@2020.12.0: +diagnostics')
 
     # Requirements for dask.delayed
     depends_on('py-cloudpickle@0.2.1:', type=('build', 'run'), when='@2,7.0: +delayed')
```

Comment:
It seems like the `+yaml` variant is no longer an option, `py-pyyaml` is always required now